rails.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ruby:2.6.4
+# RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+# RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+# RUN apt-get update -qq && apt-get install -y build-essential nodejs postgresql-client yarn
+RUN apt-get update -qq && apt-get install -y build-essential nodejs postgresql-client yarn \
+ && rm -rf /var/lib/apt/lists/* \
+ && curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Do you remember why this second direct-install may have been necessary?  Or am I misunderstanding the `apt-get` line above?

Reply:
Not necessary if you use the official yarn Debian package repo. This would be enough:

```
FROM ruby:2.6.4

RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add - \
  && echo "deb http://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list \
  && apt-get update -qq \
  && apt-get install -y build-essential nodejs postgresql-client yarn \
  && mkdir /hunterskeepers \
  && rm -rf /var/lib/apt/lists/*

WORKDIR /hunterskeepers

COPY Gemfile /hunterskeepers/Gemfile
COPY Gemfile.lock /hunterskeepers/Gemfile.lock
RUN gem install bundler:2.1.4 && bundle install
COPY . /hunterskeepers

# Rails App is Setup
```

Reply:
The only reason I can see adding in the 
> curl -o- -L https://yarnpkg.com/install.sh | bash
would be to update yarn to the latest release if the package in the distribution repository is out of date. For example Debian repos are often not the latest release of packages since the maintainers value stability and the distro is not rolling but released based. 

The curl should be removed unless the latest yarn release is necessary or if the underlying repository is out of date.

Reply:
@keigmu Delivery of Yarn packages for Linux distributions is instantaneous and in the same delivery process as the installation script except for the MSI and Chocolate/Scoop package for Windows.

Reply:
@crazy-max So does this mean the even on Debian and OpenSuse Leap which are released based distros their repositories will always have the latest stable Yarn package? If this is the case then the curl is absolutely unnecessary. Curling scripts to bash is already a security risk and we should remove code that isn't serving a purpose.

Reply:
@keigmu Yes but only if you use their package repo `http://dl.yarnpkg.com/[debian|rpm|...]`. Not sure about OpenSuse but looks like a core maintainer [takes care of it](https://build.opensuse.org/package/show/openSUSE%3AFactory/yarn).

Reply:
I think I can take a reasonable guess at the history behind the curl.  I tried running each individual command from within the ruby container, and this jumped out at me:
```
# apt-get install yarn
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'cmdtest' instead of 'yarn'
```

What I think happened:
1. `apt-get install ... yarn` was expected to install yarn properly, but was installing `cmdtest` instead
2. That output was stifled due to the `-y`
3. The `curl` was a second attempt to install it since the first wouldn't have appeared to be working
4. Things still weren't quite right so the work was committed and this PR tabled

I've removing the curl, and am using @crazy-max's suggested code, which matches yarn's installation instructions for Debian/Ubuntu: https://classic.yarnpkg.com/en/docs/install/#debian-stable

That is successfully installing `yarn`, which resolves the **yarn: error: no such option: --integrity** errors that had been happening as each service tried to start up.  It's looking really good so far, big thanks all!

Reply:
@bipster Nice! Let me know if you need a final review.

Reply:
@crazy-max No pressure, but here's the new working-for-me PR :)  https://github.com/ChaelCodes/HuntersKeepers/pull/157